modules/ROOT/partials/toolbar-button-ids/mergetags-toolbar-buttons.adoc

```diff
@@ -0,0 +1,5 @@
+[cols="1,3",options="header"]
+|===
+|Toolbar button identifier |Description
+|`+mergetags+` |Inserts a merge tag from a nested menu. The nested menu contains the merge tags list, as specified in xref:mergetags.adoc#mergetags_list[`+mergetags_list+`]. The toolbar button includes a search feature for finding specific merge tags.
```

Comment:
```suggestion
|`+mergetags+` |Menu toolbar button containing merge tags as a drop down list of nested menus and items. This list is specified by the xref:mergetags.adoc#mergetags_list[`+mergetags_list+`] option.  When a merge tag is chosen from the menu it is then inserted into the content. This button also includes a search field for finding specific merge tags.
```
The menu toolbar button can have any number of menu items, and nested menus containing menu items. The button doesn't implicitly insert a merge tag until one is chosen.